openupgrade_scripts/scripts/base_sparse_field/15.0.1.0/upgrade_analysis.txt

```diff
@@ -2,3 +2,4 @@
 ---Fields in module 'base_sparse_field'---
 ---XML records in module 'base_sparse_field'---
 ---nothing has changed in this module--
+# Nothing to do
```

Comment:
This should be done in a copied file `upgrade_analysis_work.txt`

Reply:
Thanks.
Will do